docker/build.sh

```diff
@@ -158,7 +158,7 @@ function docker_cmd()
         fi
 
         # builds the juptyer notebooks docker image that goes on dataproc clusters
-        bash ./jupyter-docker/build.sh build "${REPO}" "${DOCKER_TAG}"
+        bash ./jupyter-docker/build.sh build "gcr.io/broad-dsde-prod" "${DOCKER_TAG}"
```

Comment:
please don't hard code this value; the flags already cover this use case. If you need multiple values, then add flags to itemize this part of the build and have a default fallback to the current behavior.

To achieve this result using the script as it is today.
```
build.sh jar --registry gcr --project "broad-dsde-prod" --docker push
```

Reply:
+1. Will probably revert this and add a flag to allow pushing the notebooks image to GCR but not the Leo image.